cg/clients/freshdesk/freshdesk_client.py

```diff
@@ -71,3 +76,23 @@ def _configure_retries(session: Session) -> None:
         )
         adapter = HTTPAdapter(max_retries=retry_strategy)
         session.mount("https://", adapter)
+
+    @staticmethod
+    def create_new_ticket_attachment(content: dict, file_name: str) -> dict:
```

Comment:
This logic does not belong in the `FreshDeskAPIClient`, the client is only responsible for sending requests and parsing responses. It should not expose methods for creating attachments.